include/blmc_robots/teststand.hpp

```diff
@@ -329,6 +329,12 @@ class Teststand
    */
   Vector2d motor_max_current_;
 
+  /**
+   * @brief max_joint_torques_ (N/m)
+   * 
+   */
+  Eigen::Array2d max_joint_torques_;
```

Comment:
Why `Eigen::Array2d` instead of `Vector2d`?

Reply:
Yes we do element wise operation here, see: https://eigen.tuxfamily.org/dox/group__TutorialArrayClass.html